engine/watcher.go

```diff
@@ -51,26 +57,32 @@ func (eng *Engine) Watcher(ctx context.Context, cs *kubernetes.Clientset) {
 						default:
 							unknownNs++
 						}
+					} else {
+						wLogger.Warn().Msgf("annotation '%s' not found", eng.Options.Prefix+DesiredState)
 					}
 				}
 			}
 		}
+
 		// update metrics
-		wlLogger.Debug().Msgf("channel length: %d", wllen)
+		wLogger.Debug().Msgf("Metric - channel length: %d", wllen)
 		eng.MetricsServ.WatchlistLength.Set(float64(wllen))
 
-		wlLogger.Debug().Msgf("running namespaces: %d", runningNs)
+		wLogger.Debug().Msgf("Metric - running namespaces: %d", runningNs)
 		eng.MetricsServ.NumRunningNamspaces.Set(float64(runningNs))
 
-		wlLogger.Debug().Msgf("suspended namespaces: %d", suspendedNs)
+		wLogger.Debug().Msgf("Metric - suspended namespaces: %d", suspendedNs)
 		eng.MetricsServ.NumSuspendedNamspaces.Set(float64(suspendedNs))
 
-		wlLogger.Debug().Msgf("unknown namespaces: %d", unknownNs)
+		wLogger.Debug().Msgf("Metric - unknown namespaces: %d", unknownNs)
 		eng.MetricsServ.NumUnknownNamespaces.Set(float64(unknownNs))
 
 		eng.Mutex.Unlock()
-		wlLogger.Debug().Int("inventory_id", id).Msg("namespaces inventory ended")
-		wlLogger.Debug().Int("inventory_id", id).Msgf("inventory duration: %s", time.Since(start))
+
+		// Question: Why not add `Int("inventory_id", id)` to every log line ?
```

Comment:
:question:  how would you do this?

Reply:
The same way you do with the `namespace` in the `suspender`?

```
sLogger := eng.Logger.With().Str("routine", "suspender").Str("namespace", n.Name).Logger()
```